chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/timestamps/HistoryMessageTimestamp.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Dispatch } from "react";
+
+import { ILiveChatWidgetAction } from "../../../../../../contexts/common/ILiveChatWidgetAction";
+import { ILiveChatWidgetContext } from "../../../../../../contexts/common/ILiveChatWidgetContext";
+import { Stack } from "@fluentui/react";
+import { defaultMiddlewareLocalizedTexts } from "../../../../common/defaultProps/defaultMiddlewareLocalizedTexts";
+import { defaultTimestampContentStyles } from "../defaultStyles/defaultTimestampContentStyles";
+import { getTimestampHourMinute } from "../../../../../../common/utils";
+import useChatContextStore from "../../../../../../hooks/useChatContextStore";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const HistoryMessageTimestamp = ({ args }: any) => {
+    const [state,]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useChatContextStore();
+    const dir = state.domainStates.renderingMiddlewareProps?.timestampDir ?? state.domainStates.globalDir;
+    const {
+        activity: {
+            timestamp,
+            from: { name, role }
+        }
+    } = args;
+
+    const contentStyles = {
+        ...defaultTimestampContentStyles,
+        ...state.domainStates.renderingMiddlewareProps?.timestampContentStyleProps
+    };
+
+    const getTimeElement = (timestamp: string): string | JSX.Element => {
```

Comment:
This always returns span from what I can see, so can remove string | JSX.Element and just use JSX.Element